From 8f97b6206c7f7ed28e60e4f3c676f02e420e5cfb Mon Sep 17 00:00:00 2001
From: Himanshu Agrawal <hagraw@codeaurora.org>
Date: Thu, 23 Nov 2017 13:30:04 +0530
Subject: [PATCH 11/19] sepolicy : add secontext for eMMC blocks

Adding context to boot, recovery, cache and system block device

Change-Id: I5604c5ab842483760947d6fcb348d0723c9908b1
---
 vendor/msm8952/file_contexts | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/vendor/msm8952/file_contexts b/vendor/msm8952/file_contexts
index 6d61fc6f..98669d08 100644
--- a/vendor/msm8952/file_contexts
+++ b/vendor/msm8952/file_contexts
@@ -35,6 +35,9 @@
 /dev/block/platform/soc.0/7824900.sdhci/by-name/ssd                   u:object_r:ssd_device:s0
 /dev/block/platform/soc.0/7824900.sdhci/by-name/misc                  u:object_r:misc_block_device:s0
 /dev/block/platform/soc.0/7824900.sdhci/by-name/system                u:object_r:system_block_device:s0
+/dev/block/platform/soc.0/7824900.sdhci/by-name/recovery              u:object_r:recovery_block_device:s0
+/dev/block/platform/soc.0/7824900.sdhci/by-name/cache                 u:object_r:cache_block_device:s0
+/dev/block/platform/soc.0/7824900.sdhci/by-name/boot                  u:object_r:boot_block_device:s0
 /dev/block/platform/soc.0/7824900.sdhci/by-name/userdata              u:object_r:userdata_block_device:s0
 /dev/block/platform/soc.0/7824900.sdhci/by-name/dip                   u:object_r:dip_device:s0
 /dev/block/platform/soc.0/7824900.sdhci/by-name/mdtp                  u:object_r:mdtp_device:s0
@@ -52,6 +55,9 @@
 /dev/block/platform/soc/7824900.sdhci/by-name/ssd                   u:object_r:ssd_device:s0
 /dev/block/platform/soc/7824900.sdhci/by-name/misc                  u:object_r:misc_block_device:s0
 /dev/block/platform/soc/7824900.sdhci/by-name/system                u:object_r:system_block_device:s0
+/dev/block/platform/soc/7824900.sdhci/by-name/recovery              u:object_r:recovery_block_device:s0
+/dev/block/platform/soc/7824900.sdhci/by-name/cache                 u:object_r:cache_block_device:s0
+/dev/block/platform/soc/7824900.sdhci/by-name/boot                  u:object_r:boot_block_device:s0
 /dev/block/platform/soc/7824900.sdhci/by-name/userdata              u:object_r:userdata_block_device:s0
 /dev/block/platform/soc/7824900.sdhci/by-name/dip                   u:object_r:dip_device:s0
 /dev/block/platform/soc/7824900.sdhci/by-name/mdtp                  u:object_r:mdtp_device:s0
-- 
2.17.1

